code/modules/mob/living/simple_animal/distortion/legend/papa_bongy.dm

```diff
@@ -0,0 +1,327 @@
+	switch(icon_select)
+		if(2)
+			icon_state = "bongy_2"
+			icon_dead = "bongy_dead2"
+		if(3)
+			icon_state = "bongy_3"
+			icon_dead = "bongy_dead3"
+	..()
+
+/mob/living/simple_animal/hostile/bongy_hostile/CanAttack(atom/the_target)
+	if(ishuman(the_target))
+		var/mob/living/carbon/human/L = the_target
+		if(L.sanity_lost && L.stat != DEAD)
+			if(HAS_AI_CONTROLLER_TYPE(L, /datum/ai_controller/insane/murder/bongy))
+				return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/bongy_hostile/AttackingTarget()
+	if(!target)
+		return FALSE
+	var/mob/living/carbon/human/H = target
+	..()
+	if(ishuman(H) && H.sanity_lost)
+		BongyPanic(H)
+
+/mob/living/simple_animal/hostile/bongy_hostile/proc/BongyPanic(target)
+	var/mob/living/carbon/human/H = target
+	if(!HAS_AI_CONTROLLER_TYPE(H, /datum/ai_controller/insane/murder/bongy))
+		QDEL_NULL(H.ai_controller)
+		H.ai_controller = /datum/ai_controller/insane/murder/bongy
+		H.InitializeAIController()
+		H.apply_status_effect(/datum/status_effect/panicked_type/bongy)
+		var/obj/item/clothing/mask/facehugger/bongy/bongymask = new(get_turf(H))
+		H.equip_to_slot_if_possible(bongymask, ITEM_SLOT_MASK, FALSE, TRUE, TRUE)
+		forceMove(H)
+		death()
+		playsound(src, 'sound/distortions/papa_bongy/Chicken_Skill2_Coin2.ogg', 70, FALSE)
+		QDEL_IN(src, 5 SECONDS)
+		return
+
+/mob/living/simple_animal/hostile/bongy_hostile/throw_impact(atom/hit_atom)
+	playsound(src, 'sound/distortions/papa_bongy/Chicken_Skill2_Coin2.ogg', 70, FALSE)
+	return ..()
+
+/mob/living/simple_animal/hostile/bongy_hostile/death(gibbed)
+	if(master)
+		master.unit_list -= src
+	..()
+
+//special panic
+/datum/status_effect/panicked_type/bongy
+	icon = "murder"
+
+/datum/ai_controller/insane/murder/bongy
+	lines_type = /datum/ai_behavior/say_line/insanity_lines
+
+/datum/ai_controller/insane/murder/bongy/FindEnemies()
+	. = FALSE
+	var/mob/living/living_pawn = pawn
+	var/list/friend_list = list(
+	/mob/living/simple_animal/hostile/distortion/papa_bongy,
+	/mob/living/simple_animal/hostile/bongy_hostile,
+	)
+	var/list/potential_enemies = livinginview(9, living_pawn)
+
+	if(!LAZYLEN(potential_enemies)) // We aint see shit!
+		return
+
+	var/list/weighted_list = list()
+	for(var/mob/living/L in potential_enemies) // Oh the CHOICES!
+		if(L == living_pawn)
+			continue
+		if(L.status_flags & GODMODE)
+			continue
+		if(L.stat == DEAD)
+			continue
+		if(living_pawn.see_invisible < L.invisibility)
+			continue
+		if(L in friend_list)
+			continue
+		if(istype(L, /mob/living/simple_animal/hostile/bongy_hostile) || istype(L, /mob/living/simple_animal/hostile/distortion/papa_bongy))
+			continue
+		if(!isturf(L.loc) && !ismecha(L.loc))
+			continue
+		weighted_list += L
+	for(var/i in weighted_list)
+		if(istype(i, /mob/living/simple_animal/hostile))
+			weighted_list[i] = 3
+		else if(ishuman(i))
+			var/mob/living/carbon/human/H = i
+			if(H.sanity_lost)
+				weighted_list[i] = 0//FIXME: should team up with other panics, but doesn't
```

Comment:
If this hasn't been fixed yet, it definitely should before merge.

Reply:
I'll have to investigate this tomorrow; they shouldn't target other panicked players at all, and I think adding another condition to potential_enemies will remedy this.